Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -727,7 +734,14 @@ - (void)setContentMode:(UIViewContentMode)contentMode
 - (UIColor *)backgroundColor
 {
   _bridge_prologue_read;
-  return [UIColor colorWithCGColor:_getFromLayer(backgroundColor)];
+  if (_loaded(self)) {
+    if (_view != nil) {
+      return _view.backgroundColor;
+    } else {
+      return [UIColor colorWithCGColor:_getFromLayer(backgroundColor)];
+    }
+  }
+  return [UIColor colorWithCGColor:ASDisplayNodeGetPendingState(self).backgroundColor];
```

Comment:
This looks fine for now, assuming we'll change `_ASPendingState` to store a UIColor in a follow up diff.